components/UI/desktopNav.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FunctionComponent, MouseEvent } from "react";
+import styles from "../../styles/components/desktopNav.module.css";
+import Link from "next/link";
+import { FaDiscord, FaGithub, FaTwitter } from "react-icons/fa";
+import theme from "../../styles/theme";
+
+const DesktopNav: FunctionComponent = () => {
+  const handleClick = (e: MouseEvent) => {
+    e.stopPropagation();
+  };
+
+  return (
+    <nav onClick={handleClick} className={styles.navContainer}>
+      <div className={styles.columns}>
+        <div className={styles.column}>
+          <Link href={process.env.NEXT_PUBLIC_STARKNET_ID as string}>
+            <li className={styles.burgerItem}>Website</li>
+          </Link>
+          <Link href="https://www.starknet.id/pdfs/Terms.pdf">
+            <li className={styles.burgerItem}>Term of use</li>
+          </Link>
+          <Link href="https://docs.starknet.id/">
+            <li className={styles.burgerItem}>Documentation</li>
+          </Link>
+        </div>
+        <div className={styles.column}>
+          <Link
+            href={`${
+              process.env.NEXT_PUBLIC_STARKNET_ID as string
+            }/affiliates/individual-program`}
+          >
```

Comment:
Can you for an opening of the link in another tab instead of using href. Because people will go to another website and we also want them to stay on our website. (you can use `window.open` or use other stuff within the `Link` component).

You can do the same for all the other links